gcloud/datastore/client.py

```diff
@@ -441,8 +441,33 @@ def transaction(self):
 
     def query(self, **kwargs):
         """Proxy to :class:`gcloud.datastore.query.Query`.
-
         Passes our ``project``.
+
+        Using query to search a datastore::
```

Comment:
For virtually all subpackages we've put `Connection` and `Client` docs in the same RST file.

As for where it belongs, it's up to you.